code/game/objects/structures/crates_lockers/closets/cardboardbox.dm

```diff
@@ -48,7 +48,7 @@
 			break
 		if(Snake)
 			alerted = viewers(7,src)
-	..()
+	. = ..()
```

Comment:
shouldn't this be checked if it returns false since the above part doesn't have COMSIG_CLOSET_PRE_OPEN checks